src/scripts/parameter-chart/templates/parameters.yaml

```
@@ -0,0 +1,36 @@
+{{- if eq $prefix "" -}}
+{{- $name = $k -}}
+{{- else -}}
+{{- $name = printf "%s.%s" $prefix ($k | replace "." "\\.")  -}}
+{{- end -}}
+{{- if kindIs "map" $v }}
+{{- include "flatten" (list $v $name) }}
+{{- else if kindIs "slice" $v }}
+{{- range $i, $item := $v }}
+- name: {{ $name }}[{{ $i }}]
+    {{- if or (kindIs "bool" $item) (kindIs "float64" $item) (kindIs "int" $item) }}
+    value: {{ $item }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)